app.js

```diff
@@ -0,0 +1,133 @@
+const $ROUND_TEXT = document.querySelector(".round-text");
+const $TURN_TEXT = document.querySelector(".turn-text");
+const $PLAY_BUTTON = document.querySelector(".btn-start");
+const $START_TEXT = document.querySelector(".start-text");
+let gameStarted = false;
+let round = 0;
+//saves moves into arrays to then iterate and compare
+let CPUMoves = [];
+let HumanMoves = [];
+const $BLUE_SQUARE = document.querySelector("#blue");
+const $RED_SQUARE = document.querySelector("#red");
+const $GREEN_SQUARE = document.querySelector("#green");
+const $YELLOW_SQUARE = document.querySelector("#yellow");
+const $SQUARES = document.querySelectorAll(".square");
+
+//delay for the highlight functions
+const LIGHT_DURATION = 250;
+const MOVE_DURATION = 500;
+$PLAY_BUTTON.onclick = function () {
+  prepareStartGame();
+  startGame();
+  playRound();
+};
+
+function prepareStartGame() {
+  //hides 'press play to start' and play button
+  gameStarted = true;
+  if (gameStarted == true) {
+    $PLAY_BUTTON.classList.add("hidden");
+    $START_TEXT.classList.add("hidden");
+  }
+}
+
+function startGame() {
+  //shows text 'round x' and 'x turn'
+  $TURN_TEXT.classList.remove("hidden");
+  $ROUND_TEXT.classList.remove("hidden");
+}
+
+function computerTurn() {
+  //changes texts accordingly
+  $ROUND_TEXT.innerText = `Round ${round}`;
+  $TURN_TEXT.innerText = "CPU Turn";
+  //disables user inputs
```

Comment:
Commenting is a contentious topic, but I would say that we do not need to comment the language you are using. 
As in, someone who writes JS will know what 'enableInputs' does. 

In general I think you have a lot of comments in your code. 
Some people like that, some people will not. 
Decide which camp you are in :P